views/admin/generation/basic/misc/model.sql.php

```diff
@@ -52,8 +52,8 @@
 }
 
 ?>
-    `<?= $model['column_prefix'] ?>created_at` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
-    `<?= $model['column_prefix'] ?>updated_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    `<?= $model['column_prefix'] ?>created_at` timestamp NULL,
```

Comment:
It would be better to use the explicit syntax `NULL DEFAULT NULL` instead of the implicit syntax `NULL` for readability.